cli/cmd/upgrade.go

```diff
@@ -207,7 +207,15 @@ func getLatestKeptnRelease() (*release.Release, error) {
 		return nil, fmt.Errorf("No Keptn release found in namespace %s: %v", keptnNamespace, err)
 	}
 
-	return releases[len(releases)-1], nil
+	// iterate over releases and find the one with status = deployed
+	for _, release := range releases {
+		if release.Info.Status == "deployed" {
```

Comment:
`"deployed"` could be replaced with `release.StatusDeployed`

Reply:
Done